pkg/nutanix/client.go

```diff
@@ -270,7 +270,7 @@ type ErrorResponse struct {
 type MessageResource struct {
 
 	// Custom key-value details relevant to the status.
-	Details map[string]interface{} `json:"details,omitempty"`
+	Details interface{} `json:"details,omitempty"`
```

Comment:
Do we have to also update this https://github.com/nutanix-cloud-native/prism-go-client/blob/main/pkg/nutanix/v3/v3_structs.go#L297 ?

Reply:
Good catch. Just fixed it as well.